drivers/adc/ad7193/ad7193.c

```diff
@@ -211,34 +223,37 @@ void ad7193_reset(struct ad7193_dev *dev)
 }
 
 /***************************************************************************//**
- * @brief Set device to idle or power-down.
+ * @brief Sets device into the specified operating mode.
  *
- * @param dev      - The device structure.
- * @param pwr_mode - Selects idle mode or power-down mode.
- *                  Example: 0 - power-down
- *                           1 - idle
+ * @param dev        - The device structure.
+ * @param opt_mode	 - Operating mode to be set.
+ * @param modify_cs  - Allows Chip Select to be modified.
  *
  * @return none.
 *******************************************************************************/
-void ad7193_set_power(struct ad7193_dev *dev,
-		      uint8_t pwr_mode)
+void ad7193_set_operating_mode(struct ad7193_dev *dev,
```

Comment:
Nitpick: not necessary, but I'd take this chance to implement status return on these functions if it's not too much hassle. We should always have feedback on operations we do.

Reply:
Do you want me update all the function with status return type ?